ctx.go

```diff
@@ -244,6 +259,22 @@ var decoderPool = &sync.Pool{New: func() interface{} {
 	return decoder
 }}
 
+// SetBodyParserDecoder allow globally change the option of form decoder, update decoderPool
+func SetBodyParserDecoder(bodyParserConfig BodyParserConfig) {
```

Comment:
can we use the method for the initial write of the decoder pool ?
https://github.com/gofiber/fiber/blob/master/ctx.go#L274
@rockcreation7

Reply:
Sure, I can help update the doc on this, thanks.

Please feel free to let me know anything that can help.

Reply:
or you mean I put a change to the initial write?

Reply:
> or you mean I put a change to the initial write?
@rockcreation7 
yes exactly, with the initial set you can also use your method or ?

will merge it directly after the adjustment